### PR TITLE
pull binaries for bazel build from ci for ci-kubernetes-e2e-kubeadm-gce

### PR DESCRIPTION
**Pull request by jessicaochen**
For https://github.com/kubernetes/kubernetes/issues/59762

### CODE REVIEW

File: kubetest/anywhere.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ import (
 )
 
 const defaultKubeadmCNI = "weave"
+const defaultBuildFolder = "bazel"
```
Comment (BenTheElder) [inline kubetest/anywhere.go]: only PR builds still go here. All CI bazel builds are `gs://kubernetes-release-dev/ci/%v-bazel` and we should get this path from `gs://kubernetes-release-dev/ci/latest-bazel.txt` for CI. I'm working on a patch for this in the various places that specify it.

Comment (jessicaochen) [reply kubetest/anywhere.go]: I figured we could use the flag for a controlled switch over and then remove the flag.

Comment (BenTheElder) [reply kubetest/anywhere.go]: these jobs are all failing right now, we might as well just switch them over. I'm trying to fix CI and then sort out the presubmits. see https://github.com/kubernetes/test-infra/pull/7139

Comment (jessicaochen) [reply kubetest/anywhere.go]: Fair enough. Will close this out in favor of #7139

**Pull request closed without merging**